test-gen-2/src/LightSm.csx

```diff
@@ -1,30 +1,45 @@
 #!/usr/bin/env dotnet-script
 // This is a c# script file
 
-#r "nuget: StateSmith, 0.9.12-alpha" // this line specifies which version of StateSmith to use and download from c# nuget web service.
+#r "nuget: StateSmith, 0.9.13-alpha-tracking-expander-2" // this line specifies which version of StateSmith to use and download from c# nuget web service.
```

Comment:
The TrackingExpander changes were properly released in `0.9.14-alpha`. Feel free to use either version.